tools/cldr-apps/js/src/esm/cldrTable.js

```diff
@@ -419,7 +424,7 @@ function reallyUpdateRow(tr, theRow) {
   tr.rawValueToItem = {}; // hash:  string value to item (which has a div)
   for (let k in theRow.items) {
     const item = theRow.items[k];
-    if (item.value) {
+    if (item.value || item.value === "") {
```

Comment:
ideally encapsulate this as a boolean method -- maybe `cldrVote.hasValue(item.value)`  -- as it is the logic is duplicated half a dozen times